Android/app/src/main/java/io/github/project_travel_mate/utilities/ContributorsFragment.java

```diff
@@ -0,0 +1,184 @@
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        mContributorsView = inflater.inflate(R.layout.fragment_contributors, container, false);
+        ButterKnife.bind(this, mContributorsView);
+        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(mActivity);
+        mToken = sharedPreferences.getString(USER_TOKEN, null);
+        mHandler = new Handler(Looper.getMainLooper());
+        mAndroidContributorsAdapter = new ContributorsAdapter(this.getContext(), mAndroidContributors);
+        android_contributors_gv.setAdapter(mAndroidContributorsAdapter);
+
+        mServerContributorsAdapter = new ContributorsAdapter(this.getContext(), mServerContributors);
+        server_contributors_gv.setAdapter(mServerContributorsAdapter);
+
+        setContributors("Travel-Mate");
+        setContributors("server");
+        android_contributors_gv.setOnItemClickListener((adapterView, view, i, l) -> startActivity(
+                new Intent(Intent.ACTION_VIEW, Uri.parse(mAndroidContributors.get(i).getUrl()))
+        ));
+        server_contributors_gv.setOnItemClickListener((adapterView, view, i, l) -> startActivity(
+                new Intent(Intent.ACTION_VIEW, Uri.parse(mServerContributors.get(i).getUrl()))
+        ));
+        return mContributorsView;
+    }
+
+    private void setContributors(String repo) {
+        String uri = API_LINK_V2 + "get-contributors/" + repo;
+        Log.v("EXECUTING", uri);
+
+        //Set up client for android contributors
+        OkHttpClient client = new OkHttpClient();
+        //Request
+        final Request request = new Request.Builder()
+                .header("Authorization", "Token " + mToken)
+                .url(uri)
+                .build();
+        //Callback
+        client.newCall(request).enqueue(new Callback() {
+
+            @Override
+            public void onFailure(Call call, IOException e) {
+                Log.e("Request Failed", "Message : " + e.getMessage());
+            }
+
+            @Override
+            public void onResponse(Call call, Response response) throws IOException {
+                mHandler.post(() -> {
+                    if (response.isSuccessful() && response.body() != null) {
+                        try {
+                            final String responseString = response.body().string();
+                            Log.v("RESPONSE", responseString);
+
+                            Type listType = new TypeToken<ArrayList<Contributor>>() {
+                            }.getType();
+                            ArrayList<Contributor> contributorsList = new Gson().fromJson(responseString,
+                                    listType);
+                            updateDataAndUI(response.request().url().toString(), contributorsList);
+
+                        } catch (IOException e) {
+                            e.printStackTrace();
```

Comment:
please add logcat if needed to log the error

Reply:
I will add this too